auth:
  staff:
    owners:
      logins:
      - martikainen
      - grstray
      - robot-maps-sandbox
      groups:
      - svc_maps-duty-renderer
      - yandex_content_geodev_5685
      - svc_vops_maps
      - svc_maps-infra
      - svc_maps-core-renderer-tilesgen
labels:
  balancer_deployer_commit_id: '9401790'
  balancer_deployer_sign: c51de2bd08f00ce22c27f5b6b24a7640
  order: '50000000'
---
regexp_section:
  matcher:
    match_fsm:
      cgi: '.*experimental_datatesting=1.*'
  modules:
    - report:
        uuid: 'datatesting'
        ranges: default
    - threshold:
        lo_bytes: 734003
        hi_bytes: 838860
        pass_timeout: 10s
        recv_timeout: 1s
    - headers:
        append_func:
          X-Forwarded-For: realip
        create_func_weak:
          X-Real-IP: realip
          X-Source-Port-Y: realport
          X-Yandex-Ja3: ja3
        create:
          Host: 'core-renderer-tilesgen.testing.maps.yandex.net'
          X-Ya-Backend-Timeout-Ms: '1000'
    - balancer2:
        attempts: 3
        active:
          steady: true
          delay: '10s'
          request: 'GET /ping HTTP/1.1\nHost: core-renderer-tilesgen.testing.maps.yandex.net\r\n\r\n'
        attempts_rate_limiter:
          limit: 0.34
        generated_proxy_backends:
          proxy_options:
            connect_timeout: 70ms
            backend_timeout: 1000ms
            keepalive_count: 100
            fail_on_5xx: true
          include_backends:
            type: BY_ID
            ids: [maps_core_renderer_tilesgen_datatesting_sas,maps_core_renderer_tilesgen_datatesting_vla]
        on_error:
          errordocument:
            status: 504
            content: "Service unavailable"
